common/beerocks/bcl/CMakeLists.txt

```
@@ -4,6 +4,8 @@ project(bcl VERSION ${prplmesh_VERSION})
 set(MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
 
 find_package(PythonInterp REQUIRED)
+find_package(nl-3 REQUIRED)
```

Adding a required version is normally a good thing to do.
```suggestion
find_package(nl-3 3.2.25 REQUIRED)
```
I think it would work well with [the unversioned find_library](https://github.com/prplfoundation/prplMesh/pull/1368/commits/bab7496200354bd101ef5d0002965631d846f126#diff-3bf015b06ea8efb6ff553fcf349e0352R8) as it would probably find also the symlinks. But we need to verify this behavior.
[cmake find_library](https://cmake.org/cmake/help/latest/command/find_library.html?highlight=find_library)

---

Done.

---

I don't we should specify a specific version, as different platforms may have different versions.

---

minimum required version ensures that on these platforms the problem would be easily traced.

---

Still, it's unnecessary. We're using netlink 3 libraries here, which is enough for our needs.
We don't require a specific version.

---

So to remove it?

---

Yes